gradle/libs.versions.toml

```diff
@@ -4,7 +4,7 @@ download_task = "4.1.1"
 guava = "31.1-jre"
 jackson_xml = "2.14.1"
 launchermeta_parser = "1.0.0"
-enigma = "1.7.6"
+enigma = "1.7.5"
```

Comment:
oh, note: this PR should perhaps sync with the upstream, since we already did that update